### DISCUSSION TITLE
会有作业和考试吗？

### DISCUSSION THREAD

**Discussion by hbsun2113** [mlc-ai/mlc-zh#10] [Ideas]
如同coursera有作业要求，通过了可以发放证书？证书只是建议，作业可以帮助我们巩固复习

**Comment by Hzfengsy** [comment]
会有相应的作业（或者叫练习），证书目前暂时没有计划。感谢你的建议，我们未来可能会加以考虑

**Reply by LShady** [reply]
这周有课后练习吗？

**Reply by Hzfengsy** [reply]
本周没有，如果有课程练习会有通知
